src/components/TopBar/TopBarComponent.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import UploadIcon from 'components/TopBar/images/ic_uploads.svg';
+import SearchIcon from 'components/TopBar/images/ic_search.svg';
+import MenuIcon from 'components/TopBar/images/ic_menu.svg';
```

Comment:
what is `ic`? :)

Reply:
Will 'icon-uploads', 'icon-search' and 'icon-menu' be suitable? ^__^

Reply:
Yep :) 👍